dev-tools/globals.ts

```diff
@@ -331,6 +340,15 @@ interface EffectData extends EventMethods {
 	weather?: string
 }
 
+interface ModdedEffectData extends Partial<EffectData> {
+	effect?: Partial<ModdedEffectData>
```

Comment:
Heh, this caused a TypeScript error in TypeScript 3.2.

It looks like a TypeScript bug, but it's irrelevant, because PS itself doesn't work this way – mods shallow copy properties, they don't deep copy, so this is just `effect?: Partial<EffectData>` - which can be left off, because that's already implied by `extends Partial<EffectData>`.